Makefile

```diff
@@ -52,4 +47,11 @@ clean:
 	rm -rf dist/
 	rm -rf *.egg-info
 	find . -type d -name __pycache__ -exec rm -rf {} +
-	find . -type f -name "*.pyc" -delete
\ No newline at end of file
+	find . -type f -name "*.pyc" -delete
+
+# deploy docs
+docs-deploy:
+ifndef VERSION
+	$(error VERSION is not set)
+endif
+	$(CONDA_ENV_RUN) mike deploy --push --update-aliases $(VERSION)
```

Comment:
Nice, can the docs now be deployed with mike? That's great if we now have versioned docs - before I did it badly so only the last docs were accessible.